src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheck.java

```diff
@@ -68,6 +68,12 @@
  * </pre>
  * <ul>
  * <li>
+ * Property {@code arrayInitIndent} - Specify how far an array initialisation
```

Comment:
Not related but this should be `initialization`.

Reply:
Will do in separate PR

Reply:
@rnveach , please consider https://github.com/checkstyle/checkstyle/issues/13783